src/test/java/core/basesyntax/service/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private OperationHandler handler = new BalanceOperationHandler();
```

Comment:
```suggestion
    private OperationHandler handler;
```
services better initialize in methods annotated with before or beforeclass (depends or the unit version)